front-end/src/auth/auth0-provider-with-history.js

```diff
@@ -1,6 +1,6 @@
-import React from "react";
```

Comment:
Are these changes to auth and Login just reverting to the most recent versions from development?

Reply:
I think so - just did another pull and it looks like it's up to date.